src/Features/CSharpTest/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -9503,4 +9503,36 @@ void M()
             }
             """, index: 0, options: ImplicitTypingEverywhere());
     }
+
+    [Fact]
+    [WorkItem("https://github.com/dotnet/roslyn/issues/78204")]
+    [WorkItem("https://github.com/dotnet/roslyn/issues/77559")]
+    [WorkItem("https://github.com/dotnet/roslyn/issues/77147")]
+    public async Task TestInTopLevelStatement()
```

Comment:
@CyrusNajmabadi are the names for the tests good?

Reply:
works for me.